packages/navigator/src/browser/navigator-menu.ts

```diff
@@ -21,8 +22,28 @@ export namespace NavigatorContextMenu {
     export const DIFF = [...NAVIGATOR_CONTEXT_MENU, '5_diff'];
 }
 
+export namespace NavigatorKeybindingContexts {
```

Comment:
although `NavigatorKeybindingContexts` should stay in their own file which is named `navigator-keybinding-contexts.ts`